h2o-py/tests/testdir_algos/gbm/pyunit_gbm_wendy_problem.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import division
+from builtins import range
+import sys
+
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils, assert_equals
+from h2o.estimators import H2OGradientBoostingEstimator, H2OXGBoostEstimator
+from h2o.tree import H2OTree
+
+
+def assert_list_equals(expected, actual, delta=0):
+    assert_equals(len(expected), len(actual))
+    for i in range(0, len(expected)):
+        assert_equals(expected[i], actual[i], delta=delta)
+
+
+def gbm_predict_contributions_sorting_large():
+    fr = h2o.import_file(pyunit_utils.locate("bigdata/laptop/covtype/covtype.full.csv"))
+    # build first model with one hardware configuration    
+    ntrees=50
+    for i in range(1):
+        m = H2OGradientBoostingEstimator(seed=1234, ntrees=ntrees)
+        m.train(x=list(range(0, 12)), y="Cover_Type", training_frame=fr)
+        # pred = m.predict(fr)
+        #h2o.download_csv(pred, "/some/dictory/pred.csv"),         # build second model with same hardware configuration    
+        m2 = H2OGradientBoostingEstimator(seed=1234, ntrees=ntrees)
+        m2.train(x=list(range(0, 12)), y="Cover_Type", training_frame=fr)
+        # pred2 = m2.predict(fr)
+        #h2o.download_csv(pred2, "/some/directory/pred2.csv")        # compare prediction frames should be the same.  Basically, save the prediction as a csv and load it and compare    
+        # pred = h2o.import_file("/path/to/pred.csv")    
+        # pred2 = h2o.import_file("/path/to/pred2.csv")    
+        # for index in range(1, pred.ncols):
+        #     compare_frame_one_column(pred[index], pred2[index])
+    
+        # compare variable importance should be the same.  Can just print it out too if the two models are from different cluster runs    
+        relative_importance = m.varimp()
+        print(relative_importance)
+        relative_importance2 = m2.varimp()
+        print(relative_importance2)
+        
+        for ntree in range(ntrees):
+            for output_class in ['class_1', 'class_2', 'class_3', 'class_4', 'class_5', 'class_6', 'class_7']:
+                tree = H2OTree(model = m, tree_number = ntree, tree_class = output_class)
+                tree2 = H2OTree(model = m2, tree_number = ntree, tree_class = output_class)
+                assert_list_equals(tree.predictions, tree2.predictions)
+                assert_list_equals(tree.thresholds, tree2.thresholds, delta=1e-50) # need to specify delta to check nans
+                assert_list_equals(tree.decision_paths, tree2.decision_paths)
+                print("Tree", ntree, "class", output_class, "ok")
```

Comment:
Can you put this as a function into model_base.py and models?  Like compare GBM models?

Reply:
It would make sense to have it in pyunit_utils, imho its too specific to have it in the client code

Reply:
This makes it difficult for support people to test it when they only have h2o.jar and the python whl.  This is a legitimate function that people can use to compare two gbm models to make sure they are identical.  Instead of copying your code to our customers who want to check their gbm models every time they ask for reproducibility, we can just give them the function call.  Since model reproducibility is important to our big customers, even though it is specific, it is prudent to have the function call for them to use.  Instead of giving them your code every time they ask, we can just give them the function call.

Reply:
understand, I will do it in separate PR

Reply:
https://github.com/h2oai/h2o-3/pull/6497